Documentation update (#10)
dotkernel/light-documentation: docs/book/v1/how-tos/create-pages.md

Reviewer: Remove the below section from the bottom of the page:

> Because of the way routing works, dot (.), dash (-), underscore (_) are filtered from the `action` parameter in the routing `/page[/{action}]`.
> As a result, the `examplePageAction` function in called.

# Creating pages

## Register the page route

Open `config/autoload/local.php` and locate the **routes** key.
Under the **page** key you will find an associative array with a couple of already existing routes.
You create a new route by appending a new line to that array:

```php
'example-page' => 'example-template',
```

> > The array key is the page URI, the array value is the template name used by that page.

Reviewer: ```suggestion
> The array key is the page slug, the array value is the template name used by that page.
```
Since my review on the `routing.md` file, calling this slug instead of URI seems to make more sense because it refers only to the `about` part, not the whole path `/page/about`.

> In order to be displayed this new `example-template` can then be added to the `src/Page/templates/page/example-template.html.twig` in the following way as an example:

Reviewer: ```suggestion
In order to be displayed this new `example-template` needs to be added to the website's navigation area.
To do this, open `src/App/templates/layout/default.html.twig`, locate the `ul` inside the `div` with id `navbarHeader` and append the below code to it:
```

> ## The `action` function

Reviewer: ```suggestion
## Register the page route
```

> The first step is to add the new pages in `config/Autoload/local.php`.

Reviewer: ```suggestion
Open `config/autoload/local.php` and locate the **routes** key.
Under the **page** key you will find an associative array with a couple of already existing routes.
You create a new route by appending a new line to that array:
'example-page' => 'example-template',

> The array key is the page URI, the array value is the template name used by that page.
```
Wrap the new line to be added inside PHP code markers, so that it shows up correctly in the documentation.

> This means adding a `Route` for each page, as seen below.

Reviewer: ```suggestion
```

> ```php

Reviewer: Remove this PHP code because the `routes` array is already there.
See the [current version](https://github.com/dotkernel/light/blob/1.0/config/autoload/local.php.dist#L18) of the file.
